tvplayer/src/main/java/com/kaltura/tvplayer/TVPlayerType.java

```diff
@@ -0,0 +1,7 @@
+package com.kaltura.tvplayer;
+
+public enum TVPlayerType {
```

Comment:
You don't think this should be an inner class of KalturaPlayer? There it can be called just `Type`, so when you use it from outside the class it's `KalturaPlayer.Type`.